azure-pipelines.yml

```diff
@@ -9,45 +9,36 @@ pr:
 - rel/*
 
 pool:
-  vmImage: windows-2022
+  vmImage: windows-latest
+
+variables:
+  Build.Configuration: Release
+  DotNet.Tools: $(Agent.ToolsDirectory)/dotnet
+  DotNet.Version: 7.0.x
 
 jobs:
 - job: BuildBits
+  displayName: Build and Test solution
   timeoutInMinutes: 60
   steps:
 
-  # Install the .NET Core 3.1 SDK
-  - task: UseDotNet@2
-    displayName: Install the .NET Core 3.1 SDK
-    inputs:
-      version: 3.1.x
-
-  # Install the .NET 6 SDK
-  - task: UseDotNet@2
-    displayName: Install the .NET 6 SDK
+  # Cache .NET SDKs and Tools across pipeline runs
+  - task: Cache@2
+    displayName: Cache .NET SDKs
     inputs:
-      version: 6.0.x
+      key: 'dotnet | "$(Agent.OS)" | "$(DotNet.Version)"'
+      restoreKeys: |
+        dotnet | "$(Agent.OS)"
+        dotnet
+      path: $(DotNet.Tools)
 
   # Install the .NET 7 SDK
   - task: UseDotNet@2
     displayName: Install the .NET 7 SDK
     inputs:
-      version: 7.0.x
+      version: $(DotNet.Version)
```

Comment:
We're no longer installing the .NET Core 3.1 SDK, is that just included in the VM image?

Reply:
Yep.